app/views/partials/_jobs.html.erb

```diff
@@ -13,12 +13,22 @@
       <td class="index-job-detail">
         <h5><%= job.posting_date.strftime("%m/%d") %></h5>
       </td>
-      <td>
+      <% if current_user.jobsliked.include?(job) %>
```

Comment:
We had to add this conditional so that when you refresh/visit the jobs#index it reflects the jobs belonging to a user and not the buttons again